testfiles/Fault_HumidistatOffset_Supermarket.idf

```diff
@@ -2431,8 +2431,10 @@
 	
   FaultModel:HumidistatOffset, 
     Humi_Offset_SalesFloor, !- Name
+    SalesFloor Humidistat,  !- Humidistat Name
+    ThermostatOffsetIndependent, !- Humidistat Offset Type
     ,  				  		!- Availability Schedule Name
     , 				  		!- Severity Schedule Name
     5,        	 	  		!- Reference Humidistat Offset
```

Comment:
@mjwitte 
@Myoldmopar told me the IDF texts would be re-processed (e.g., notes would be aligned). I guess the unit can be added too. Anyway, I will add the units in the IDFs.

I used "tabs" to align all the new added texts, and they look aligned in Notpad++. But when I open them in Github, they look unaligned.